### PR TITLE
Update links to flow components

### PR DESCRIPTION
**Pull request by MithunKinarullathil**
- Update all links to opensource components.

- [x] Make sure you are opening from a **topic/feature/bugfix branch**
- [x] Ensure that the PR title represents the desired changes
- [x] Ensure that the PR description detail the desired changes
- [ ] Link to relevant issues in GitHub or Jira
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue

[^note]:
    Put an `x` into the [ ] to show you have filled the information.
    The template comes from https://github.com/MOV-AI/.github/blob/master/.github/pull_request_template.md
    You can override it by creating .github/pull_request_template.md  in your own repository

### CODE REVIEW

**Comment by AlexFernandes-MOVAI** [comment]
nice one, thanks ;)

**Comment by AlexFernandes-MOVAI** [comment]
Also, I believe we should start to have a dedicated installation.md to shorten a bit our main README.md

**Comment by MithunKinarullathil** [comment]
> Also, I believe we should start to have a dedicated installation.md to shorten a bit our main README.md

https://github.com/MOV-AI/movai-flow/issues/155

**Review by AlexFernandes-MOVAI — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -138,14 +138,14 @@ Installing the MOV.AI Flow™ package will provide a few command line tools to c
...
- - spawner: Flow orchestrator of ROS nodes
- - haproxy: web proxy
- - simulator: Ignition Fortress container with graphical capabilities enabled
- - ros-tools: rviz container with graphical capabilities enabled
+ - [redis-master](https://github.com/MOV-AI/containers-redis2): Master DB of the cluster
+ - [redis-local](https://github.com/MOV-AI/containers-redis2): Local DB of the cluster
+ - [backend](https://github.com/MOV-AI/backend): Web service application
+ - [ros-master](https://github.com/MOV-AI/containers-ros-master): ROS core service
+ - [spawner](https://github.com/MOV-AI/containers-spawner-base): Flow orchestrator of ROS and MOV.AI nodes
+ - [haproxy](https://github.com/MOV-AI/containers-haproxy): Web proxy
```
[the remark this answers is not in the document]
Comment (MithunKinarullathil) [reply README.md:146]: Is it this one? https://github.com/haproxy/haproxy
I don't see the 2.4.15 tag there in the repository.

[the remark this answers is not in the document]
Comment (MithunKinarullathil) [reply README.md:146]: @AlexFernandes-MOVAI ?

[the remark this answers is not in the document]
Comment (AlexFernandes-MOVAI) [reply README.md:146]: https://hub.docker.com/_/haproxy

**Review by AlexFernandes-MOVAI — APPROVED** [review]

**Pull request merged**